vmbindings/dummyvm/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "mmtk_dummyvm"
 version = "0.0.1"
 authors = [" <>"]
+edition = "2021"
```

Comment:
Any specific reason to add edition 2021 here? `mmtk-core` is using edition `2018` (though we should migrate to 2021).

Reply:
I used the logging facilities (`info!(...)`, `debug!(...)`, ...) in dummyvm and found the macros need to be imported differently than `mmtk-core`.  [The rules for importing macros changed in 2018 edition.](https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#macros)  I had two options:

1.  Keep it as 2015, and add `use` statements in the 2015 style.
2.  Add `edition=2018` or `edition=2021` to `Cargo.toml`.

I think we will eventually migrate to a new edition, so there is pointless to stick to 2015.  For 2018 vs 2021, Rust is designed so that [changing the edition of one crate does not affect another crate](https://doc.rust-lang.org/edition-guide/editions/index.html#editions-do-not-split-the-ecosystem).  So I skipped 2018 and set it to 2021 directly.